packages/react/src/components/Dropdown/index.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Dropdown } from '..';
+import { Item } from '..';
+
+const defaultProps = () => ({
```

Comment:
Yeah you don't need this, you can just add an `id` for each instance
And the action should exist in the `Playground` story

Reply:
👍